src/main/java/mate/academy/springboot/swagger/controller/ProductController.java

```diff
@@ -0,0 +1,121 @@
+                                            @RequestParam(defaultValue = "20")
+                                            Integer count,
+                                            @ApiParam(value = "page number", defaultValue = "0")
+                                            @RequestParam(defaultValue = "0")
+                                            Integer page,
+                                            @ApiParam(value = "sort param", defaultValue = "price")
+                                            @RequestParam(defaultValue = "price")
+                                            String sortBy) {
+        Sort sort = Sort.by(SortUtil.parse(sortBy));
+        PageRequest request = PageRequest.of(page, count, sort);
+        return productService.findAll(request).stream()
+                .map(responseMapper::mapToDto)
+                .collect(Collectors.toList());
+    }
+
+    @GetMapping("/by-price")
+    @ApiOperation("Endpoint to get all products where price is between two values received as a "
+            + "RequestParam inputs, also with pagination and ability to sort by price or by title "
+            + "in ASC or DESC order")
+    public List<ProductResponseDto> findAllByPriceBetween(@ApiParam(value = "price from value",
+            defaultValue = "0")
+                                                          @RequestParam(defaultValue = "0")
+                                                          BigDecimal from,
+                                                          @ApiParam(value = "price to value",
+                                                                  defaultValue = "1000")
+                                                          @RequestParam(defaultValue = "1000")
+                                                          BigDecimal to,
+                                                          @ApiParam(value = "page size",
+                                                                  defaultValue = "20")
+                                                          @RequestParam(defaultValue = "20")
+                                                          Integer count,
+                                                          @ApiParam(value = "page number",
+                                                                  defaultValue = "0")
+                                                          @RequestParam(defaultValue = "0")
+                                                          Integer page,
+                                                          @ApiParam(value = "sort param",
+                                                                  defaultValue = "id")
+                                                          @RequestParam(defaultValue = "id")
+                                                          String sortBy) {
+        Sort sort = Sort.by(SortUtil.parse(sortBy));
```

Comment:
```suggestion
        Sort sort = Sort.by(SortUtils.parse(sortBy));
```
Please rename this util class correspondingly